### DISCUSSION TITLE
GRPCError:RPC  failed,the  engine    might   have  crashed:rpc_run_step_impl:failed  with   error  code   StatusCode.UNKNOWN,details:Exception   calling   application:local  variable   response   refereneced  before   assignment

### DISCUSSION THREAD

**Discussion by yzjg** [alibaba/GraphScope#475] [Q&A]
GRPCError:RPC  failed,the  engine    might   have  crashed:rpc_run_step_impl:failed  with   error  code   StatusCode.UNKNOWN,details:Exception   calling   application:local  variable   response   refereneced  before   assignment

**Comment by sighingnow** [comment]
May I know the version of graphscope that you are working with?

**Reply by yzjg** [reply]
graphScope  version  is       0.5.0,k8s  is     1.16.4 ,this     graphScope  is    installed  by      helm   charts  that     included   in  the     graphScope   Source  directory,so   please    help   me  now,I  don't  know  what  this   happening

**Reply by sighingnow** [reply]
Do you try to load a graph or try to run an algorithm when the error occurs?

Is there any further detailed error message or exception backtrace?

**Reply by yzjg** [reply]
![image](https://user-images.githubusercontent.com/25189806/124570911-e176cb00-de79-11eb-8671-4af9d2231084.png)

**Reply by yzjg** [reply]
and  just  now,i     add  http_proxy   env  var,then    the  session     throw    exception
![image](https://user-images.githubusercontent.com/25189806/124572938-a6759700-de7b-11eb-9d26-1dc50961490e.png)

**Reply by yzjg** [reply]
this  is    totally   offically   example,  but  i  can't  run  normally,i   want  to    know  why and  what  happening

**Reply by yzjg** [reply]
![image](https://user-images.githubusercontent.com/25189806/124572887-99f13e80-de7b-11eb-9607-2bfb97290217.png)

**Reply by sighingnow** [reply]
Could you please provide a snapshot of the logs before the error?

**Reply by yzjg** [reply]
we deploy    k8s  env   in  corporate    network  that  cann't   connect  to  the   www,
so   i    can    only    caputer    picture    outside  
these   show   entire   code   and  process  
![image](https://user-images.githubusercontent.com/25189806/124580308-9319fa00-de82-11eb-81f6-b3838d689cdd.png)
![image](https://user-images.githubusercontent.com/25189806/124580344-99a87180-de82-11eb-9d78-e08e37afd59b.png)
![image](https://user-images.githubusercontent.com/25189806/124580370-9f9e5280-de82-11eb-8d45-c1068c04eca6.png)
![image](https://user-images.githubusercontent.com/25189806/124580400-a6c56080-de82-11eb-9023-a08abfb38c07.png)
![image](https://user-images.githubusercontent.com/25189806/124580467-bb095d80-de82-11eb-8f3e-394f194dda4b.png)
![image](https://user-images.githubusercontent.com/25189806/124580480-bf357b00-de82-11eb-8351-f53055d0fc0f.png)
![image](https://user-images.githubusercontent.com/25189806/124580495-c52b5c00-de82-11eb-814b-05950989e3ec.png)
![image](https://user-images.githubusercontent.com/25189806/124580509-c9577980-de82-11eb-9627-67a237681191.png)
![image](https://user-images.githubusercontent.com/25189806/124580526-cd839700-de82-11eb-9445-e585d4f33de2.png)
![image](https://user-images.githubusercontent.com/25189806/124580545-d2484b00-de82-11eb-8f1a-43e19cc983a6.png)

**Comment by lidongze0629** [comment]
Hi, @yzjg, 

**Connect to the service delpoyed by Helm**
If you deploy GraphScope with [Helm](https://artifacthub.io/packages/helm/graphscope/graphscope), you should use `sess = graphscope.session(addr='xxxx')` to connect to the service from client.
